features/templates.feature

```diff
@@ -28,54 +29,77 @@ Feature: Templates
 
     Rules:
         - Template owner can publish a template if
-              - The format of template is valid
-              - The build that is used to publish has the right permissions
-              - No existing version already exists
+            - The format of template is valid
+            - The build that is used to publish has the right permissions
+            - No existing version already exists
         - Publishing a template should be stored to be retrieved later.
         - User can use a template by specifying it in their job. If they specify
-              - test/template:            pulls the latest version of test/template
-              - test/template@1:          pulls the latest version of major 1 for test/template
-              - test/template@1-stable:   pulls the latest version of major 1 with label "stable" for test/template
-              - test/template@1.2:        pulls the latest patch of 1.2 of test/template
+            - test/template:            pulls the latest version of test/template
+            - test/template@1:          pulls the latest version of major 1 for test/template
+            - test/template@1-stable:   pulls the latest version of major 1 with label "stable" for test/template
+            - test/template@1.2:        pulls the latest patch of 1.2 of test/template
 
     Scenario Outline: A template owner wants to validate their template
         Given a <template> job-level template
-        When they submit it to the API
+        When they submit it to the validator
         Then they are notified it has <number> errors
 
         Examples:
-          | template | number |
-          |  valid   |   no   |
-          |  invalid |  some  |
-
-    @ignore
-    Scenario Outline: Merges template into user's config
-        Given an existing pipeline with job main
-        And an existing template called Test
-        And user specifies template Test in job main
+            | template | number |
+            | valid    | no     |
+            | invalid  | some   |
+
+    Scenario Outline: Publish a template
+        Given a "<template>" template
+        When a pipeline with the "right" permission "<status>" to publish the template in "<job>"
+        Then the template "<stored>" stored
+
+        Examples:
+            | template      | status  | job             | stored |
+            | publish-test  | SUCCESS | publish-test    | is     |
+            | test-template | FAILURE | publish-invalid | is not |
+
+    Scenario Outline: Publish a template by another pipeline
+        Given a "test-template" template
+        And the template exists
+        When a pipeline with the "wrong" permission "FAILURE" to publish the template in "<job>"
```

Comment:
```suggestion
        When a pipeline with the "wrong" permission fails to publish the template in "<job>"
```
this reads more easily for me in English, but you would probably need to set up something to map `fails => FAILURE` and `succeeds=>SUCCESS` if you make the change

Reply:
This one seems to be better, so I'll fix it.
Thank you.